src/components/Reports/PeopleReport/PeopleReport.jsx

```diff
@@ -27,7 +27,7 @@ import { ReportPage } from '../sharedComponents/ReportPage';
 import { getPeopleReportData } from './selectors';
 import { PeopleTasksPieChart } from './components';
 import ToggleSwitch from '../../UserProfile/UserProfileEdit/ToggleSwitch';
-import { Checkbox } from '../../common/Checkbox';
+import  {Checkbox} from '../../common/Checkbox';
```

Comment:
I believe this change is not needed and should stay as: `import { Checkbox } from '../../common/Checkbox';`

I would try installing ESLint and Prettier extensions in your IDE if you are using VS Code.